pom.xml

```diff
@@ -201,13 +201,17 @@
       <version>2.6.1</version>
     </dependency>
 
-    <!-- https://mvnrepository.com/artifact/org.springframework.security.oauth.boot/spring-security-oauth2-autoconfigure -->
     <dependency>
-      <groupId>org.springframework.security.oauth.boot</groupId>
-      <artifactId>spring-security-oauth2-autoconfigure</artifactId>
-      <version>2.6.1</version>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-oauth2-client</artifactId>
+      <version>5.6.0</version>
```

Comment:
extract common spring-security version (for client, jose, maybe for other security dependecies also)